solr/core/src/java/org/apache/solr/handler/replication/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** V2 API implementations for "replication" APIs. */
+package org.apache.solr.handler.replication;
```

Comment:
[0] If you want to create a package specifically for the APIs we're pulling out of ReplicationHandler, that's fine by me.  But:

1.  Could you add 'api' to the package name somewhere?  (e.g. `o.a.s.handler.api.replication`). That's the convention I've tried to follow elsewhere.
2. Do you envision each ReplicationHandler API/command its own class, or do you imagine multiple of them going into CoreReplicationAPI?  If the former, maybe we should rename CoreReplicationAPI to be more specific (e.g. `GetIndexVersionAPI`?). If the latter, maybe we don't need a separate package for this code after all?